include/ilang/ila/ast/expr_op.h

```diff
@@ -179,11 +179,47 @@ class ExprOpSub : public ExprOp {
                      const std::string& suffix = "") const;
 }; // class ExprOpSub
 
-// TODO ExprOpDiv
+/// \brief The wrapper for unsigned division.
+class ExprOpDiv : public ExprOp {
+public:
+  /// Constructor for DIV operation.
+  ExprOpDiv(const ExprPtr arg0, const ExprPtr arg1);
+  std::string op_name() const { return "DIV"; }
+  z3::expr GetZ3Expr(z3::context& ctx, const Z3ExprVec& expr_vec,
+                     const std::string& suffix = "") const;
+}; // class ExprOpDiv
+
+// \brief The wrapper for signed remainder
```

Comment:
Double slash comment will not be recognized by Doxygen. Use triple comments instead.